Light up bot/README.md

```diff
@@ -0,0 +1,46 @@
+# Light up bot
+
+This script configures an object detection robot using Viam. Hardware that are required: your computer with a webcam, a Kasa Smart Plug, and a desk lamp. This robot will be able to turn the lights on/off when it detects a person in front of it.
+
+As you get and set up the vision service params, you have to change the “model_path” to where you tflite package lives, and the “label_path” to where your text file lives.
+
+It also expects the smart plug to be attached to the light source (desk lamp in this case), and the following robot configuration in Viam app:
+
+``` json
+{
+  "components": [
+     {
+      "depends_on": [],
+      "model": "webcam",
+      "name": "camera",
+      "type": "camera",
+      "attributes": {
+        "debug": false,
+        "height_px": 0,
+        "path_pattern": "",
+        "width_px": 0,
+        "intrinsic_parameters": {
```

Comment:
You can take these and distortion_parameters out since they are not set, not required for this project and might confuse people